### PR TITLE
add seed data

### PR DESCRIPTION
**Pull request by turino**
seed data is 11.7 MB

could switch to csv format if we want to save space

### CODE REVIEW

**Comment by turino** [comment]
runs in under 30 sec on my machine

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+connection = ActiveRecord::Base.connection
+
+%w[states districts zip_codes districts_zip_codes legislators].each do |table|
+  sql = File.read("db/seed_data/#{table}.sql")
+  statements = sql.split(/;$/)
+
+  ActiveRecord::Base.transaction do
```
Comment (Rio517) [inline db/seeds.rb]: Did you try `ActiveRecord::Base.connection.execute(IO.read("db/seed_data/#{table}.sql"))`?  That would avoid ruby having to parse the individual lines and probably speed it up a little.  

This merge request looks good.  I don't love manually assigning the sequences, but unless we wanted to make a lot of changes and create all the records w/ rails, I cannot think of a better way to do it. 

I'm going to go ahead and merge.  Maybe try that out the one liner above if you have time late.

File: db/seed_data/districts.sql [hunk 2]
```diff
@@ -0,0 +1,441 @@
+INSERT INTO districts (id, district, state_id, created_at, updated_at) VALUES (31, '17', 6, '2015-02-06 00:48:06.415016', '2015-02-06 00:48:06.416166');
+INSERT INTO districts (id, district, state_id, created_at, updated_at) VALUES (32, '18', 6, '2015-02-06 00:48:06.421442', '2015-02-06 00:48:06.42334');
+INSERT INTO districts (id, district, state_id, created_at, updated_at) VALUES (35, '20', 6, '2015-02-06 00:48:06.435113', '2015-02-06 00:48:06.436709');
```
Comment (Rio517) [inline db/seed_data/districts.sql]: Actually,  I don't think we  even need the ID field here.  The table is already created and the sequence rules are set.  Wouldn't postgres take care of ID assignment for us automatically?  

Not worth revisiting now, but maybe some other point.

Comment (turino) [inline db/seed_data/districts.sql]: We're inserting state_id, district_id, and zip_code_id into other tables, so it's essential that they match the id value from the db on my system that the data was copied from. We could could skip id field on legislators, though

Comment (Rio517) [inline db/seed_data/districts.sql]: Great point. I hadn't considered that.  Don't worry about it in that case.

File: db/seeds.rb (hunk 1 above)
Comment (turino) [inline db/seeds.rb]: Worked really well, actually! Now takes half the time.

**Comment by turino** [comment]
@Rio517 
Warning: I deleted some migrations (sorry!). You can get up to date by manually dropping table campaigns_legislators. Feel free to reject and I'll add the migrations back. I just couldn't stand to leave the migrations for two different join tables that I created and then dropped...

**Pull request merged**